rosidl_typesupport_introspection_c/resource/srv__type_support.c.em

```diff
@@ -58,10 +70,13 @@ static rosidl_service_type_support_t @(function_prefix)__@(service.namespaced_ty
 
```

Comment:
`service_introspection_message__create` and `service_introspection_message__destroy` need to be implemented and added to `rosidl_service_type_support_t` above to complete the implementation

Reply:
Done in https://github.com/ros2/rosidl/pull/700/commits/f07ad3027ce2c1a854b938b81da7ecb311571f97